packages/istanbul-lib-report/lib/summarizer.js

```diff
@@ -4,98 +4,99 @@
  */
 'use strict';
 
-const util = require('util');
 const coverage = require('istanbul-lib-coverage');
 const Path = require('./path');
 const tree = require('./tree');
 const BaseNode = tree.Node;
 const BaseTree = tree.Tree;
 
-function ReportNode(path, fileCoverage) {
-    this.path = path;
-    this.parent = null;
-    this.fileCoverage = fileCoverage;
-    this.children = [];
-}
+class ReportNode extends BaseNode {
+    constructor(path, fileCoverage) {
+        super();
 
-util.inherits(ReportNode, BaseNode);
+        this.path = path;
+        this.parent = null;
+        this.fileCoverage = fileCoverage;
+        this.children = [];
+    }
 
-ReportNode.prototype.addChild = function(child) {
-    child.parent = this;
-    this.children.push(child);
-};
+    addChild(child) {
+        child.parent = this;
+        this.children.push(child);
+    }
 
-ReportNode.prototype.asRelative = function(p) {
-    /* istanbul ignore if */
-    if (p.substring(0, 1) === '/') {
-        return p.substring(1);
+    asRelative(p) {
+        /* istanbul ignore if */
```

Comment:
why are we ignoring this if?

Reply:
I just checked, the `if` path is not hit.  I'll remove the ignore though as I don't think it's impossible to test for this.